### PR TITLE
front: remove document panel & fix chip style

### LINKED ISSUE #5220: Link and document in the frontend is not used and implemented
This is useless currently:

Image

Version: cb78a63b

### LINKED ISSUE #5308: Adding tags/labels to a study leads to a tiny scrollbar in the study card
### What happened?

I created a study with 3 tags. Everything went fine but when i go back to my project page i see the card of the study like this, with a tiny scrollbar caused by the tags :

image

### What did you expect to happen?

As long as there's only one line with tags there should not be a scroll because it's supposed to fit.

### How can we reproduce it (as minimally and precisely as possible)?

1. Create a study with tags
2. Go back to the parent project
3. See bug.

### What operating system, browser and environment are you using?

- **Browser**: Chrome
- **OS**: Windows 11
- **Env**: staging (internal)

### OSRD version (top right corner `Account` button > `Informations`)

e32e129131ada79b92704fc7ca6bf137888e8489

### PR DESCRIPTION
**Pull request by clarani**
closes #5220 
closes #5308 

- remove document panel
- fix chip style (try to add a tag to a study or project)

image

### CODE REVIEW

**Review by Wadjetz — APPROVED** [review]

**Review by RomainValls — APPROVED** [review]

File: front/src/applications/operationalStudies/views/Study.tsx [hunk 1]
```diff
@@ -275,7 +223,7 @@ export default function Study() {
                 />
               </div>
               <div className="row">
-                <div className="col-xl-9 col-lg-8 col-md-7">
+                <div className="col-xl-12">
```
Comment (nicolaswurtz) [inline front/src/applications/operationalStudies/views/Study.tsx]: Avoid `-12` without necessity, just remove the two `divs` and we'll be good, thx !

File: front/src/styles/scss/_overrideBootstrapSNCF.scss [hunk 2]
```diff
@@ -69,8 +69,8 @@
 .form-control-container .chips-input {
   width: auto !important;
   flex-grow: 1;
-  min-height: 2.45rem;
   line-height: 1;
+  outline: none;
```
Comment (nicolaswurtz) [inline front/src/styles/scss/_overrideBootstrapSNCF.scss]: Why ? We should never remove outline, it's needed for accessibility https://www.a11yproject.com/posts/never-remove-css-outlines/

**Review by nicolaswurtz — CHANGES_REQUESTED** [review]

File: front/src/applications/operationalStudies/views/Study.tsx [hunk 3]
```diff
@@ -274,50 +222,47 @@ export default function Study() {
...
-                              studyName={study.name}
-                              state={state}
-                              done={idx <= studyStates.indexOf(study.state as StudyState)}
-                              tags={study.tags}
-                            />
-                          )
-                      )}
-                    </div>
-                  )}
+              <div className="row flex-column p-2">
```
Comment (nicolaswurtz) [inline front/src/applications/operationalStudies/views/Study.tsx]: No row without col after ;-)

**Review by nicolaswurtz — CHANGES_REQUESTED** [review]

**Review by nicolaswurtz — APPROVED** [review]

File: front/src/styles/scss/_overrideBootstrapSNCF.scss (hunk 2 above)
Comment (clarani) [reply front/src/styles/scss/_overrideBootstrapSNCF.scss]: OK I removed this line, it was to remove the green border which is not very fancy, but it's ok :)

**Pull request merged**